lib/src/rocket.rs

```diff
@@ -148,6 +148,7 @@ impl Rocket {
 
             let mut form_items = FormItems(form);
             if let Some(("_method", value)) = form_items.next() {
+                if req.method() != Method::Post { return; }
```

Comment:
This is a great catch! The correct change is to amend line 144 in rocket.rs so that it reads:

```rust
        if req.method() == Method::Post && req.content_type().is_form() && data_len >= min_len {
```

Don't change this line.